VMEncryption/main/patch/redhatPatching.py

```diff
@@ -95,7 +95,11 @@ def install_cryptsetup(self):
             packages.remove('cryptsetup')
             
         if self.command_executor.Execute("rpm -q " + " ".join(packages)):
-            self.command_executor.Execute("yum install -y " + " ".join(packages))
+            return_code = self.command_executor.Execute("yum install -y " + " ".join(packages), timeout=30)
+            if return_code == -9:
+                msg = "Command: yum install timed out. Make sure yum is configured correctly and there are no network problems."
```

Comment:
RHEL BYOS images must be registered before yum can be used.  Not sure if that warrants a mention here, it would probably fail with a different exit code in that case.

Reply:
This exit code only captures timeout. As of now we do not capture other failures during package installation. I will try and address the broader issue of failing early if dependent packages are not installed in a different PR.